package.json

```diff
@@ -14,7 +17,8 @@
   },
   "homepage": "https://github.com/scality/Arsenal#readme",
   "dependencies": {
-    "ajv": "4.1.5",
+    "ajv": "^4.11.2",
+    "eslint-plugin-react": "^4.3.0",
```

Comment:
in npm 3, the peer dependencies are not automatically installed anymore, so we need this plugin :)